project-generator/src/main/java/io/syndesis/project/converter/DefaultProjectGenerator.java

```diff
@@ -104,9 +123,31 @@ public DefaultProjectGenerator(ConnectorCatalog connectorCatalog, ProjectGenerat
         });
 
         Map<String, byte[]> contents = new HashMap<>();
+
+        for (Templates.Resource additionalResource : generatorProperties.getTemplates().getAdditionalResources()) {
+            String overridePath = generatorProperties.getTemplates().getOverridePath();
+            URL resource = null;
+
+            if (!Strings.isEmpty(overridePath)) {
+                resource = getClass().getResource("templates/" + overridePath + "/" + additionalResource.getSource());
```

Comment:
ok, got it. I think it would be sufficient if we add the overridePath to an IOException if thrown. I think for debugging its enough if we add a comment top-level on the generated files for what they are for (e.g. via XML / properties comments in the templates)